### PR TITLE
update icewind/smb to 1.0.0

### PR DESCRIPTION
**Pull request by icewind1991**
Includes fixes from @mmattel 

cc @DeepDiver1975 @PVince81

### CODE REVIEW

File: apps/files_external/3rdparty/icewind/smb/example.php [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<?php
```
Comment (MorrisJobke) [inline apps/files_external/3rdparty/icewind/smb/example.php]: `example.php`? is this really needed?

**Comment by MorrisJobke** [comment]
Are there issues that are fixed by this?

File: apps/files_external/3rdparty/icewind/smb/example.php (hunk 1 above)
Comment (icewind1991) [inline apps/files_external/3rdparty/icewind/smb/example.php]: removed

**Comment by icewind1991** [comment]
@owncloud-bot retest this please

**Comment by icewind1991** [comment]
> Are there issues that are fixed by this?

smblcient not always being closed: https://github.com/icewind1991/SMB/pull/26
stat result not always being parsed correctly using non-standard servers: https://github.com/icewind1991/SMB/pull/25

**Comment by mmattel** [comment]
Sorry for the  7.0.7-next-maintenance milestone, thick fingers on my iPad...
I removed it alrady

**Comment by Xenopathic** [comment]
:+1:

**Comment by MorrisJobke** [comment]
:+1:

**Pull request merged**